synapse/config/key.py

```diff
@@ -95,9 +95,49 @@ def read_config(self, config, config_dir_path, **kwargs):
 
         # if neither trusted_key_servers nor perspectives are given, use the default.
         if "perspectives" not in config and "trusted_key_servers" not in config:
+            logger.warn(
+                """
+                Synapse requires that a list of trusted key servers are
+                specified in order to provide signing keys for other servers in
+                the federation.
+
+                This homeserver does not have a trusted key server configured in
+                homeserver.yaml and will fall back to the default of
+                'matrix.org'.
+
+                Trusted key servers should be long lived and stable which
+                makes matrix.org a good choice for many admins, but some admins may
+                wish to choose another. To suppress this warning, the admin
+                should set 'trusted_key_servers' in homeserver.yaml to their
+                desired key server.
```

Comment:
and `suppress_key_server_warning`?

Reply:
Yes-ish - only if matrix.org is specified

Reply:
Have edited to make clearer so that admins only need to change once